components/list/list-item-generic.js

```diff
@@ -0,0 +1,73 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemGeneric extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
+				display: grid;
+				grid-template-columns:
+					[start outside-control-start] minmax(0, min-content)
+					[control-start outside-control-end] minmax(0, min-content)
+					[control-end content-start] auto
+					[content-end actions-start] auto
+					[end actions-end];
+			}
+			::slotted([slot="outside-control"]),
+			::slotted([slot="control"]),
+			::slotted([slot="content"]),
+			::slotted([slot="actions"]) {
+				grid-row: 1 / 2;
+			}
+			::slotted([slot="outside-control"]) {
+				grid-column: outside-control-start / outside-control-end;
+			}
+
+			::slotted([slot="control"]) {
+				grid-column: control-start / control-end;
+			}
+
+			::slotted([slot="content"]) {
+				grid-column: content-start / content-end;
+			}
+
+			::slotted([slot="actions"])  {
```

Comment:
This one might be good to do add `position:relative` so then anything in this will have a z-index relative to the one defined here. We should make sure this is actually how it works...

Reply:
I'll double check that.

Reply:
Since this is grid z-index this might not matter :D